app/controllers/albums.js

```diff
@@ -60,3 +60,21 @@ exports.listAlbums = (req, res, next) =>
       }
     })
     .catch(next);
+
+exports.listAlbumsPhotos = (req, res, next) => {
+  const albumId = parseInt(req.params.id);
+  return getEmailFromToken(req.body.token)
+    .then(email => User.findUserByEmail(email))
+    .then(user =>
+      Purchase.findPurchase(user.id, albumId).then(foundPurchase => {
+        if (foundPurchase) {
+          return requestAlbumPhotos(albumId).then(albumPhotos => {
+            logger.info(`Listing albums photos of album with id ${albumId}`);
```

Comment:
This log is good, but also remember to always add a log at the start of the request. Something like `Attempting to list album photos of ...`